src/main/java/com/github/vkorobkov/jfixtures/processor/Processor.java

```diff
@@ -45,14 +45,16 @@ private void processFixture(Fixture fixture) {
     private void handleFixtureInstructions(Fixture fixture) {
         val tableName = fixture.name;
         val table = context.getConfig().table(tableName);
+        val cleanMethod = table.getCleanMethod();
+
         log.info("Processing table '" + tableName + "'");
         List<Instruction> fixtureInstructions = new ArrayList<>();
 
         fixtureInstructions.addAll(table.getBeforeCleanup().stream()
                 .map(s -> customSql(tableName, s)).collect(Collectors.toList()));
 
-        if (CleanMethod.DELETE == table.getCleanMethod()) {
-            fixtureInstructions.add(cleanupTable(fixture));
+        if (CleanMethod.DELETE == cleanMethod || CleanMethod.TRUNCATE == cleanMethod) {
+            fixtureInstructions.add(cleanupTable(fixture, table.getCleanMethod()));
```

Comment:
What do you think if we _always_ add a `CleanupTable` instruction even when the `CleanMethod` is `NONE`? But in case of `NONE` the instruction won't add any SQL

Reply:
Agree